client/state/timezones/actions.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+import {
+	TIMEZONES_RECEIVE,
+	TIMEZONES_REQUEST,
+	TIMEZONES_REQUEST_FAILURE,
+	TIMEZONES_REQUEST_SUCCESS
+} from 'state/action-types';
+
+export const timezonesRequestAction = () => ( {
+	type: TIMEZONES_REQUEST
+} );
+
+export const timezonesRequestSuccessAction = () => ( {
+	type: TIMEZONES_REQUEST_SUCCESS
+} );
+
+export const timezonesRequestFailureAction = error => ( {
+	type: TIMEZONES_REQUEST_FAILURE,
+	error
+} );
+
+/*
+ * The data given in as the function parameter has already been checked
+ * from the `timezones` data-layer.
+ * Take a look to requestTimezones function for more information.
+ */
+export const timezonesReceiveAction = ( data ) => ( {
```

Comment:
related to the note above, shall we think about this action truly as "we received timezone data" or something more akin to "we want to add this timezone data into the app"

the name may be relevant